src/failed-spec-parser.js

```diff
@@ -2,8 +2,34 @@ export default function (output = '') {
   let match = null
   let CUCUMBERJS_TEST = /^\d+ scenarios?/m
   let failedSpecs = new Set()
+  let PROTRACTOR_SHARDED = /------------------------------------/g
```

Comment:
I believe using multiple browsers also outputs these lines. I'm not sure if there's another way of detecting this. We should double check that it doesn't interfere with non-sharded multi browser tests